sources/docker-compose.yml

```diff
@@ -150,6 +150,7 @@ services:
       - ESDC_ENVIRONMENT_CODE=${ESDC_ENVIRONMENT_CODE}
       - APPLICATION_ARCHIVE_DAYS=${APPLICATION_ARCHIVE_DAYS}
       - QUEUE_CONSUMERS_PORT=${QUEUE_CONSUMERS_PORT}
+      - IS_FULLTIME_ALLOWED=${IS_FULLTIME_ALLOWED}
```

Comment:
Why do we need it in the queue consumers? I am sorry if I missed it from some earlier discussion in the meetings. 

We are getting the value for `IS_FULLTIME_ALLOWED` from the api as below:

<img width="734" alt="image" src="https://github.com/bcgov/SIMS/assets/7859295/b673855e-64a8-41b9-bde4-64dfb08fa9fb">

Please let me know if I am missing something.

Reply:
The portion of code that you shared in on queue consumers NOT api and queue consumers is missing the ENV variable.

Reply:
Thanks for the clarification on the call @dheepak-aot